packages/rs-drive-abci/src/query/data_contract_based_queries/data_contract/v0/mod.rs

```diff
@@ -17,38 +18,27 @@ use dpp::version::PlatformVersion;
 impl<C> Platform<C> {
     pub(super) fn query_data_contract_v0(
         &self,
-        state: &PlatformState,
-        request: GetDataContractRequestV0,
+        GetDataContractRequestV0 { id, prove }: GetDataContractRequestV0,
         platform_version: &PlatformVersion,
-    ) -> Result<QueryValidationResult<Vec<u8>>, Error> {
-        let metadata = self.response_metadata_v0(state);
-        let quorum_type = self.config.validator_set_quorum_type() as u32;
-        let GetDataContractRequestV0 { id, prove } = request;
+    ) -> Result<QueryValidationResult<GetDataContractResponseV0>, Error> {
```

Comment:
I believe this should return a GetDataContractResponse and not a GetDataContractResponseV0

Reply:
This is versioned function that accept GetDataContractRequestV0, so I guess it does make sense It returns the GetDataContractResponseV0 and parent function deal with request and response versioning.